### PR TITLE
PMM-6608 Added more logs

### PR DESCRIPTION
**Pull request by percona-csalguero**
- Added error logs instead of stop collecting metrics in case of errors.
- Added options to disable getDiagnosticData and replicasetGetStatus collectors

### CODE REVIEW

File: exporter/diagnostic_data_collector.go [hunk 1]
```diff
@@ -43,15 +43,14 @@ func (d *diagnosticDataCollector) Collect(ch chan<- prometheus.Metric) {
 	res := d.client.Database("admin").RunCommand(context.TODO(), cmd)
 
 	if err := res.Decode(&m); err != nil {
-		ch <- prometheus.NewInvalidMetric(prometheus.NewInvalidDesc(err), err)
```
Comment (BupycHuk) [inline exporter/diagnostic_data_collector.go:46]: Why can't we do both?
Log error and return invalid metric?

File: main.go [hunk 2]
```diff
@@ -84,20 +87,26 @@ func main() {
 
 	if opts.URI == "" {
 		opts.URI = os.Getenv("MONGODB_URI")
```
Comment (BupycHuk) [inline main.go:89]: kong let's to override value by using `env` tag https://github.com/alecthomas/kong#supported-tags.

**Review by BupycHuk — CHANGES_REQUESTED** [review]

File: exporter/diagnostic_data_collector.go (hunk 1 above)
Comment (percona-csalguero) [reply exporter/diagnostic_data_collector.go:46]: If we do that, prometheus will stop receiveing metrics so, we are preventing all other metrics to be exposed

Comment (BupycHuk) [reply exporter/diagnostic_data_collector.go:46]: got it

**Review by BupycHuk — APPROVED** [review]

**Review by AlekSi — APPROVED** [review]
:shipit:

**Pull request merged**